signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/azure/KeyImporter.java

```diff
@@ -12,30 +12,16 @@
  */
 package tech.pegasys.signers.secp256k1.azure;
 
-import java.math.BigInteger;
-import java.util.Arrays;
-
-import com.google.common.collect.Lists;
-import com.google.common.io.BaseEncoding;
-import com.microsoft.azure.keyvault.KeyVaultClientCustom;
-import com.microsoft.azure.keyvault.webkey.JsonWebKey;
-import com.microsoft.azure.keyvault.webkey.JsonWebKeyCurveName;
-import com.microsoft.azure.keyvault.webkey.JsonWebKeyOperation;
-import com.microsoft.azure.keyvault.webkey.JsonWebKeyType;
-import org.web3j.crypto.ECKeyPair;
-
 public class KeyImporter {
 
   // Creates a SECP256k1 key (usable for Ethereum signing) in Azure KeyVault.
   public static void importKeyToCloudVault() {
+    /*
```

Comment:
I am pretty sure this class is not used anymore, may be deleted if not referenced anywhere??

Reply:
It wasn't used before - was helpful for personal usage, but is now removed.